frontend/debian/cuttlefish-orchestration.cuttlefish-host_orchestrator.init

```diff
@@ -39,7 +39,7 @@ orchestrator_https_port=${orchestrator_https_port:-1443}
 orchestrator_tls_cert_dir=${orchestrator_tls_cert_dir:-"/etc/cuttlefish-common/host_orchestrator/cert"}
 orchestrator_instance_manager_enabled=${orchestrator_instance_manager_enabled:-false}
 orchestrator_android_build_url=${orchestrator_android_build_url:-"https://androidbuildinternal.googleapis.com"}
-orchestrator_cvdbin_android_build_id=${orchestrator_cvdbin_android_build_id:-"8687975"}
+orchestrator_cvdbin_android_build_id=${orchestrator_cvdbin_android_build_id:-"9802869"}
```

Comment:
why is the version specified in two different places? could we get away with only one?

Reply:
yes, this is a practice we've been following for most of the config parameters. I'll create a bug to address this.